src/renderer/src/views/Home/Settings/ProjectConfig/InviteDeviceModal.tsx

```diff
@@ -0,0 +1,140 @@
+import { defineMessages, useIntl } from 'react-intl'
+import styled from '@emotion/styled'
+import { Dialog, DialogActions, DialogContent, DialogTitle, useTheme } from '@mui/material'
+import LaunchIcon from '@mui/icons-material/Launch'
+import WifiIcon from '@mui/icons-material/Wifi'
+import { Column, Row } from '@renderer/components/LayoutComponents'
+import { spacing } from '@renderer/theme/spacing'
+
+import { Text } from './Text'
+import { Button } from './Button'
+import { PeerCard } from './PeerCard'
+import { PressableText } from './PressableText'
+import { useMapeoDeviceNonMembersListIds } from '@renderer/hooks/useMapeoDeviceList'
+import { useMapeoDevice } from '@renderer/hooks/useMapeoDevice'
+import { Device } from '@renderer/hooks/stores/mapeoDeviceStore'
+
+const NonMemberPeerCard = ({ id, onClick }: { id: string; onClick: (d: Device) => void }) => {
+  const { device } = useMapeoDevice(id)
+  return (
+    <PeerCard
+      title={device.name}
+      subtitle={device.deviceId}
+      deviceType={device.deviceType}
+      onClick={() => onClick(device)}
+    />
+  )
+}
+
+export const InviteDeviceModal = ({ open, onClose }: { open: boolean; onClose: () => void }) => {
+  const { formatMessage: t } = useIntl()
+  const theme = useTheme()
+
+  const nonMemberDevices = useMapeoDeviceNonMembersListIds()
+
+  return (
+    <Dialog
+      open={open}
+      onClose={(_, reason) => {
+        if (reason === 'backdropClick' || reason === 'escapeKeyDown') return
+        onClose()
+      }}
+      fullScreen
+      sx={{ margin: '48px' }}
+    >
+      <Row
+        borderBottom={`1px solid ${theme.grey.light}`}
+        padding={spacing.large}
+        justifyContent="space-between"
+        alignItems="center"
+      >
+        <DialogTitle fontWeight="600" fontSize="24px" sx={{ padding: 0 }}>
+          {t(m.title)}
+        </DialogTitle>
+        <Row>
+          {/* TODO: Use actual Wifi info */}
+          <WifiIcon sx={{ marginInlineEnd: spacing.small }} />
+          <Text size="small" fontWeight="500" color={theme.blue.dark}>
+            SpicyWifi - 5G
+          </Text>
```

Comment:
I think we can leave as is since we have less than a week a other tickets still.

Can take out comment

Reply:
addressed via https://github.com/digidem/mapeo-desktop-shell/pull/45/commits/9d0d96694cba3aad38b86b820f8fdab21219d9a6